tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
 ---
 layout: tutorial-beginner
+title: Installing Extension
 ---
 
 {% include base.html %}
 
-# To be done
+# Installing an Extension
```

Comment:
Now that you mention it, you are totally right. Let's hear what @kaikreuzer thinks about that. Sure seems like we should agree on one term.